nilearn/image/resampling.py

```diff
@@ -86,7 +86,7 @@ def from_matrix_vector(matrix, vector):
     return t
 
 
-def coord_transform(x, y, z, affine):
+def coord_transform(x, y, z, affine, squeeze=True):
```

Comment:
> > -def coord_transform(x, y, z, affine):
> > +def coord_transform(x, y, z, affine, squeeze=True):

> Should this be the default behavior ?

I don't think so. It's always annoying to have to deal with a number of
dimensions that can fluctuate.